### ISSUE TITLE
Cache does not support pagination?

### ISSUE THREAD

**Issue by daltonrooney** [vaersaagod/seomate#52]
I have a category archive, and I'd like the meta title to be "Category Title: Page 1", "Category Title: Page 2", etc, depending on the current pagination. I'm passing in the title via entry context, then loading in my template like:

```
{% set seomate = {
  meta: {
    title: article_index_context.pageSeoTitle,
  }
} %}
```

This works correctly when the SEOMate cache is disabled but not when the cache is turned on.

Would it be possible to disable caching on a per-template basis like this: 
```
{% set seomate = {
  cacheEnabled: false,
  meta: {
    title: article_index_context.pageSeoTitle,
  }
} %}
```

Or somehow update the cache key to support page numbers?

**Comment by mmikkel** [comment]
Thanks a lot for the PR @daltonrooney. Going to get that fix into the next release.

> Would it be possible to disable caching on a per-template basis

[Sure is!](https://github.com/vaersaagod/seomate#overriding-meta-data-and-settings-from-your-templates) Just make sure to add any config settings you want to override to a `config` key in your `seomate` variable:

```twig
{% set seomate = {
    meta: {
        ...
    },
    config: {
       cacheEnabled: false
    }
} %}
```

**Comment by mmikkel** [comment]
Fixed in [SEOMate 1.1.10](https://github.com/vaersaagod/seomate/releases/tag/v1.1.10)
